crates/stdarch-test/src/lib.rs

```diff
@@ -119,6 +119,9 @@ pub fn assert(shim_addr: usize, fnname: &str, expected: &str) {
                 // it from the slightly more restrictive 22 instructions below.
                 "cpuid" => 30,
 
+                // It takes at least 23 because of the many moves, also there are padding NOPs in x86
+                "aesencwide128kl" | "aesdecwide128kl" | "aesencwide256kl" | "aesdecwide256kl" => 29,
```

Comment:
Rather than special casing padding nops, we could change the test suite to ignore any instructions after `ret`.